### PR TITLE
Ft/hero section Reviews implementation

### PR DESCRIPTION
**Pull request by Ntwalii**
Slightly resized the green shape and lowered the text as mentioned by the QA.

### CODE REVIEW

File: src/sections/hero.html [hunk 1]
```diff
@@ -9,13 +9,13 @@
...
         <img src="./src/images/red.svg" alt="red shape" class="z-[4] lg:z-[3] absolute bottom-[4.7%] md:bottom-[8.2%] right-[5%] lg:left-[13.04%] lg:top-[49.58%] w-[73.5%] lg:w-[47.66%]"/>
         <img src="./src/images/blue.svg" alt="blue shape" class="z-[3] lg:z-[5] absolute bottom-[11%] md:bottom-[19%] lg:top-[6.7%] w-[70%] lg:w-[46.4993395%] right-[5%] lg:left-[41.22%]"/>
         <img src="./src/images/yellow.svg" alt="yellow shape" class="z-[2] absolute left-0 bottom-[7.5%] md:bottom-[12%] lg:top-0 lg:left-0 w-[76%] lg:w-[47.95%]"/>
         <img src="./src/images/right-green.svg" alt="Green shape" class="z-[-10] absolute bottom-0 lg:right-0 lg:top-[24.42%] right-0 w-[42%] lg:w-[26.82%] "/>
         <img src="./src/images/pink.svg" alt="Pink shape" class="absolute bottom-[26%] md:bottom-[43%] right-[9%] w-[30%] lg:w-[15.59%] lg:left-[39.56%] lg:top-[2.27%]"/>
-        <div id="paragraphDiv" class="absolute z-30 inset-0 lg:flex justify-center lg:items-center md:hidden top-[60%] lg:top-0">
+        <div class="absolute z-30 inset-0 lg:flex justify-center lg:items-center md:hidden top-[58%] lg:top-[30%]">
```
Comment (josueahadi) [inline src/sections/hero.html]: The text is too far down on the large screen breakpoint.

**Review by josueahadi — CHANGES_REQUESTED** [review]

**Review by gloriaumutoni — CHANGES_REQUESTED** [review]
minimize the padding top for the words on the hero section

File: src/sections/hero.html (hunk 1 above)
Comment (Ntwalii) [reply src/sections/hero.html]: Resolved

**Review by gloriaumutoni — APPROVED** [review]

**Review by beylar — APPROVED** [review]

**Review by josueahadi — APPROVED** [review]

**Review by Kingcedru — APPROVED** [review]
done

**Pull request merged**